etc/profile-m-z/transmission-cli.profile

```diff
@@ -10,5 +10,7 @@ include globals.local
 private-bin transmission-cli
 private-etc @tls-ca
 
+whitelist /usr/share/transmission
+
```

Comment:
Suggestion: Move it above `private-` commands as per profile.template.

Reply:
Thx, moved it.